### DISCUSSION TITLE
What happens when I delete "_remotely-save-metadata-on-remote.json" and then sync the vault?

### DISCUSSION THREAD

**Discussion by zhechenghu** [remotely-save/remotely-save#99] [General]
Thank you a lot for this awesome plugin at first!!!

Sometimes my sync fails (see #90 ). And I found that if I delete the file "_remotely-save-metadata-on-remote.json" in remote, the plugin will work again. And It seems there is no data loss. I wonder what happens when I do such a thing? Thanks a lot!

**Comment by fyears** [comment]
1. the request expired porblem should be (hopefully) fixed in next version
2. it you delete some files on device A, remotely save stores those file names in the json file and upload it, so that the plugin on device B can download the json file and know which files should also be deleted on device B. if you delete the json file on s3, then the plugin on device B doesn’t know what files should be deleted, and potentially may reupload some files to s3. So yes no files will be lost, instead, some files might be reuploaded.

**Reply by zhechenghu** [reply]
OK, I understand. Thank you for your reply. Looking forward to the next version!
